### PR TITLE
Removed TOS checkbox and validation from signup page

### LINKED ISSUE #928: Sign-up: Update Styling for screen 1 - Basic Info
Should look like this:

image

Not like this:

image

# Acceptance Criteria:

- [ ] Update styling to match design
- [ ] Add "why we ask this" button
- [ ] Remove TOS checkbox (handled elsewhere), including dialog and associated redux action
- [ ] Restore header (not sure when it stopped working)

# Related Issues

- User can upload replacement for avatar: #179 (US)
- Remove birth_region, birth_city. Add nationality #737   (US)

### PR DESCRIPTION
**Pull request by gsidebo**
#### What are the relevant tickets?

Closes #928 

#### What's this PR do?

Removed the TOS checkbox from the signup flow

#### How should this be manually tested?

Set `filled_out` to False for your user and go through the signup flow. Make sure you don't see any errors related to TOS.

#### Any background context you want to provide?

The nationality bit (part of the issue description) was taken care of in a different issue

### CODE REVIEW

**Comment by aliceriot** [comment]
If I set `filled_out` to `False` I can save the profile on the first 3 steps, but I get a 400 when I try to save from the privacy page.

Without setting `filled_out` to `False` I also got a 400 on the first profile step.

Any thoughts?

**Comment by aliceriot** [comment]
nvm, my fault >_<

oops

**Comment by aliceriot** [comment]
functionality looks good, reading code now

File: static/js/constants.js [hunk 1]
```diff
@@ -143,7 +142,6 @@ export const ELASTICSEARCH_RESPONSE = {
 export const USER_PROFILE_RESPONSE = {
   "username": SETTINGS.username,
   "filled_out": true,
-  "agreed_to_terms_of_service": true,
```
Comment (aliceriot) [inline static/js/constants.js]: this is still included on profiles when we get a response from the API, is there a particular reason we want to delete this here?

File: static/js/flow/profileTypes.js [hunk 2]
```diff
@@ -55,7 +54,6 @@ export type Profile = {
   country: string;
   state_or_territory: string;
   email_optin: boolean;
-  agreed_to_terms_of_service: boolean;
```
Comment (aliceriot) [inline static/js/flow/profileTypes.js]: same question here, looks like this hasn't yet been deleted from the API

**Comment by aliceriot** [comment]
just two quick questions! back to you

**Comment by gsidebo** [comment]
@aliceriot that `edx_level_of_education` error kinda gives me the willies. i ran into that too. if some error like that occurs for some actual user, they will only see that general error page, and i don't think we have monitoring set up to alert us when that happens in production. i feel like we could have potential sign-up errors in the app right now, and we wouldn't know about them when they happen

**Comment by noisecapella** [comment]
The `edx_level_of_education` error was fixed recently: https://github.com/mitodl/micromasters/pull/972

**Comment by gsidebo** [comment]
@noisecapella good news, and i think it's worth our time to see if there are other required fields that a user can't actually change in signup

**Comment by aliceriot** [comment]
we should definitely get some monitoring set-up for production errors for this kind of thing, at the least.

**Comment by pdpinch** [comment]
I added an [issue for setting up monitoring](https://github.com/mitodl/micromasters/issues/980). Feel free to edit or comment with specifics. 

FYI @mitodl/devops

**Comment by aliceriot** [comment]
LGTM :+1: 🚢 :shipit:

**Pull request merged**